src/app/frontend/deploy/deploy.scss

```diff
@@ -18,3 +18,38 @@
   // TODO(bryk): Find a better, application wide constant for the min width.
   min-width: 600px;
 }
+
+.kd-deploy-label-list {
```

Comment:
Styling using only stock Angular Material would be great. I'm still learning this so every workaround to get rid of this will be good for me.